en/more-08.md

```diff
@@ -2,6 +2,194 @@
 title: "More on: Tables"
 ---
 
+
+## The other preamble contents
+
+As the lesson didn't cover all the available preamble-tokens I'll go on and
+explain a few others here. You might want to revisit the tables at the start of
+the lesson to get an overview of the things available. I hope the short
```

Comment:
We don't use first person elsewhere.

Reply:
I've gone for 'we' in various places, to keep the language informal

Reply:
yep, I meant to add that if it is used, then use the plural we form not "I"